### PR TITLE
VEN-1242 | Customer Page: Order Details View

### PR DESCRIPTION
**Pull request by Nizar-Rahme**
## Description :sparkles:
- Add a modal to display single order details 
- Support actions for a single order
- Add pagination to the Orders table
- Add filter to the Orders table based on the status
- Delete unused components

### Automated tests :gear:️
- Updated all relevant unit tests and snapshots

### Manual testing :construction_worker_man:
- Go to a single customer page
- The new invoice card should have pagination and a filter for filtering the table based on the status
- By clicking on one of the orders, a modal opens showing detailed information 
- A dropdown at the top of the modal shows all the actions that can be taken on the order in question

## Screenshots :camera_flash:
image

### CODE REVIEW

File: src/features/customerView/invoiceCard/InvoiceCard.tsx [hunk 1]
```diff
@@ -41,33 +49,39 @@ const selectedInvoicesAtom = atom<{ [x: string]: Invoice }>({
 
 const InvoiceCard = ({ berthLeases, className, customer, invoiceData, refetchQueries }: InvoiceCardProps) => {
   const { t, i18n } = useTranslation();
+  const [openInvoice, setOpenInvoice] = useState<Invoice | null>(null);
 
   const { selectedRows, selectedRowIdsDict, onSelectionChange } = usePreserveSelect<Invoice>(selectedInvoicesAtom);
 
-  const statues = invoiceData
+  const openInvoiceStatues = openInvoice ? [{ order: openInvoice.status, lease: openInvoice.lease.status }] : undefined;
```
Comment (TatuArvela) [inline src/features/customerView/invoiceCard/InvoiceCard.tsx:56]: Minor typo, should be `openInvoiceStatuses`

File: src/features/customerView/invoiceCard/InvoiceCard.tsx [hunk 2]
```diff
@@ -41,33 +49,39 @@ const selectedInvoicesAtom = atom<{ [x: string]: Invoice }>({
...
+  const selectedInvoicesStatues = invoiceData
```
Comment (TatuArvela) [inline src/features/customerView/invoiceCard/InvoiceCard.tsx:57]: Same typo here, statues -> statuses

File: src/features/customerView/invoiceModal/InvoiceModal.tsx [hunk 3]
```diff
@@ -6,105 +6,167 @@ import { isBerthInvoice, isWinterStorageInvoice } from '../utils';
...
+            <InvoiceActions
+              className={styles.select}
+              selectedAction={selectedAction}
+              actions={[
+                {
+                  value: actions.sendOffer.value,
+                  label: actions.sendOffer.label,
+                  disabled: actions.sendOffer.disabled,
+                  onClick: actions.sendOffer.onSelect,
+                },
```
Comment (TatuArvela) [inline src/features/customerView/invoiceModal/InvoiceModal.tsx:55]: I think we could shrink these lines down somehow, but that's not strictly required

**Review by TatuArvela — APPROVED** [review]
Great job! Added a few comments to consider

File: src/features/customerView/invoiceModal/InvoiceModal.tsx (hunk 3 above)
Comment (Nizar-Rahme) [reply src/features/customerView/invoiceModal/InvoiceModal.tsx]: Done! Now, these are slightly shorter.

**Pull request merged**